tests/reporting/test_market_unit.py

```diff
@@ -451,9 +439,94 @@ def test_market_migrate_through_one_fork(localFixture, initializeMarket, constan
     assert initializeMarket.getLastReportersDisputeBond() == longToHexString(0)
     assert initializeMarket.getTentativeWinningPayoutDistributionHash() == initializeMarket.getDesignatedReportPayoutHash()
 
+def test_market_disavow_tokens(localFixture, initializeMarket, mockUniverse, mockReportingWindow, mockMarket, mockStakeTokenFactory):
```

Comment:
`initializeMarket` feels like a bad name for this fixture.  Any reason we aren't just calling it `market`?  Do you perhaps mean `initializedMarket`?

Reply:
agreed, I'll change to `initializedMarket`